src/components/molecules/TableComponent/TableComponent.tsx

```diff
@@ -40,18 +40,21 @@ const TableComponent: React.FunctionComponent<TableComponentPropsType> = ({
 
         {usersSeatedAtTable &&
           usersSeatedAtTable.length >= 0 &&
-          usersSeatedAtTable.map((user) => (
-            <img
-              onClick={() => setSelectedUserProfile(user)}
-              key={user.id}
-              className="profile-icon table-participant-picture"
-              src={user.pictureUrl || DEFAULT_PROFILE_IMAGE}
-              title={user.partyName}
-              alt={`${user.partyName} profile`}
-              width={imageSize}
-              height={imageSize}
-            />
-          ))}
+          usersSeatedAtTable.map(
```

Comment:
This one might mistakenly make people think a table isn't full when it is, perhaps we should show the default avatar and the default party name for a taken table seat?